python/pyspark/sql/pandas/map_ops.py

```diff
@@ -60,6 +60,7 @@ def mapInPandas(
         schema : :class:`pyspark.sql.types.DataType` or str
             the return type of the `func` in PySpark. The value can be either a
             :class:`pyspark.sql.types.DataType` object or a DDL-formatted type string.
+        isBarrier : Use barrier mode execution if True.
```

Comment:
We should mark these new arguments by `.. versionadded`?

Reply:
Yes we should. with the directive `.. versionchanged`.

Reply:
let me just make a quick followup.

Reply:
https://github.com/apache/spark/pull/40571